From 7c640d272ba563b3a61634c386052b530697b9a3 Mon Sep 17 00:00:00 2001
From: David Heidelberg <david@ixit.cz>
Date: Wed, 20 May 2026 17:43:35 +0200
Subject: [PATCH 1/5] Partial revert "Renamed cz.nic.datovka.mobile.desktop
 file."

Breaks clean flatpak install.

This reverts commit 6f07cc90e8a1071efdcf49b963ae9d50761d3ddf.

Signed-off-by: David Heidelberg <david@ixit.cz>
---
 ....cz.mobile-datovka.desktop => cz.nic.datovka.mobile.desktop} | 0
 mobile-datovka.pro                                              | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename linux/{nic.cz.mobile-datovka.desktop => cz.nic.datovka.mobile.desktop} (100%)

diff --git a/linux/nic.cz.mobile-datovka.desktop b/linux/cz.nic.datovka.mobile.desktop
similarity index 100%
rename from linux/nic.cz.mobile-datovka.desktop
rename to linux/cz.nic.datovka.mobile.desktop
diff --git a/mobile-datovka.pro b/mobile-datovka.pro
index 8775b0ce..0673ae17 100644
--- a/mobile-datovka.pro
+++ b/mobile-datovka.pro
@@ -560,17 +560,17 @@ linux {
     BINDIR="$${PREFIX}/bin"
     DATADIR="$${PREFIX}/share"
 
     LOCALE_INST_DIR = "$${DATADIR}/$${APP_NAME}/localisations"
 
     target.path = "$${BINDIR}"
 
     desktop.path = "$${DATADIR}/applications"
-    desktop.files += "linux/nic.cz.mobile-datovka.desktop"
+    desktop.files += "linux/cz.nic.datovka.mobile.desktop"
 
     metainfo.path = "$${DATADIR}/metainfo"
     metainfo.files += "linux/cz.nic.datovka.mobile.metainfo.xml"
 
     icon64.path = "$${DATADIR}/icons/hicolor/64x64/apps"
     icon64.files += "res/datovka.png"
 
     localisation.path = "$${LOCALE_INST_DIR}"
-- 
2.53.0

